tests/Unit/Date/DateFormatterFactoryTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DR\Internationalization\Tests\Unit\Date;
+
+use DR\Internationalization\Date\DateFormatOptions;
+use DR\Internationalization\Date\DateFormatterFactory;
+use IntlDateFormatter;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\TestCase;
+
+#[CoversClass(DateFormatterFactory::class)]
+class DateFormatterFactoryTest extends TestCase
+{
+    public function testDefault(): void
+    {
+        $emptyOptions = new DateFormatOptions('NL', 'Europe/Stockholm');
```

Comment:
https://github.com/123InktJules/internationalization/pull/1/files#diff-49663d2ad11ce5b72c7e2ac1c0bebf9c85c840d62a70b373fd77a4006458d25fR25

Isn't that an invalid local notation?
Should be nl_NL or nl I think